Platform/ElkhartlakeBoardPkg/Library/Stage2BoardInitLib/UpdateAcpiGnvs.c

```diff
@@ -537,9 +537,10 @@ PlatformUpdateAcpiGnvs (
   PchNvs                  = (PCH_NVS_AREA *) &GlobalNvs->PchNvs;
   CpuNvs                  = (CPU_NVS_AREA *) &GlobalNvs->CpuNvs;
   SaNvs                   = (SYSTEM_AGENT_NVS_AREA *) &GlobalNvs->SaNvs;
-  FeaturesCfgData         = (FEATURES_CFG_DATA *) FindConfigDataByTag(CDATA_FEATURES_TAG);
   ZeroMem (GlobalNvs, sizeof (GLOBAL_NVS_AREA));
   SiCfgData = (SILICON_CFG_DATA *)FindConfigDataByTag (CDATA_SILICON_TAG);
+  FeaturesCfgData = (FEATURES_CFG_DATA *) FindConfigDataByTag (CDATA_FEATURES_TAG);
+  ASSERT (FeaturesCfgData != NULL);

```

Comment:
A default value to continue to boot if FeaturesCfgData is not found instead of assertion.

Reply:
done removed.